src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3573,12 +3573,18 @@ void MethodContext::recGetThreadLocalStaticBlocksInfo(CORINFO_THREAD_STATIC_BLOC
     Agnostic_GetThreadLocalStaticBlocksInfo value;
     ZeroMemory(&value, sizeof(value));
 
-    value.tlsIndex.handle                   = CastHandle(pInfo->tlsIndex.addr);
-    value.tlsIndex.accessType               = pInfo->tlsIndex.accessType;
-    value.offsetOfMaxThreadStaticBlocks     = pInfo->offsetOfMaxThreadStaticBlocks;
-    value.offsetOfThreadLocalStoragePointer = pInfo->offsetOfThreadLocalStoragePointer;
-    value.offsetOfThreadStaticBlocks        = pInfo->offsetOfThreadStaticBlocks;
-    value.offsetOfGCDataPointer             = pInfo->offsetOfGCDataPointer;
+#ifdef TARGET_WINDOWS
```

Comment:
Do we need these ifdefs? Can't we just copy all the data no matter what?

Reply:
If the VM (or caller?) zeros the structure before filling it in, the non-used portions will be appropriately zero.